### PR TITLE
OPS-2653 Fix copy/paste munge in az field

### PR DESCRIPTION
**Pull request by flybd5**
The definition of the az field got munged and mixed up with a region definition. Fixed, tested, works now.

### CODE REVIEW

File: tasks/create_vpc_subnet.yml [hunk 1]
```diff
@@ -34,8 +34,8 @@
     # Looked-up values
     vpc_id: "{{ aws_vpc_subnet_vpc_id }}"
     cidr: "{{ subnet.cidr }}"
-    # Optional values
-    az: "{{ subnet.region | default(aws_vpc_subnet_default_region) }}{{ subnet.az | default(omit) }}"
+    # Optional values
```
Comment (cytopia) [inline tasks/create_vpc_subnet.yml]: Trailing whitespace after comment

File: tasks/create_vpc_subnet.yml [hunk 2]
```diff
@@ -34,8 +34,8 @@
     # Looked-up values
     vpc_id: "{{ aws_vpc_subnet_vpc_id }}"
     cidr: "{{ subnet.cidr }}"
-    # Optional values
-    az: "{{ subnet.region | default(aws_vpc_subnet_default_region) }}{{ subnet.az | default(omit) }}"
+    # Optional values    
+    az: "{{ subnet.az | default(omit) }}"
```
Comment (cytopia) [inline tasks/create_vpc_subnet.yml]: Really good catch. :+1: This is also the reason to consume our own roles for CI tests of other AWS stuff.

File: tasks/create_vpc_subnet.yml (hunk 1 above)
Comment (flybd5) [reply tasks/create_vpc_subnet.yml]: Removed

**Comment by flybd5** [comment]
Tagged v.1.0.1

**Pull request merged**